src/vue/modules/issuance-explorer/components/issuances-table.vue

```diff
@@ -64,12 +72,14 @@
 <script>
 import EmailGetter from '@/vue/common/EmailGetter'
 import NoDataMessage from '@/vue/common/NoDataMessage'
+import RequestStateViewer from '@modules/requests/shared/components/request-state-viewer'
```

Comment:
Modules should not use components of other modules. Modules can use common application-wide components. Otherwise modules are useless.

We should copy-paste the `request-state-viewer` in the current module instead of reusing one, the module does not know nothing about.